docs/shared/getting-started/installation.md

```diff
@@ -90,4 +90,121 @@ pnpm install --global nx@latest
 
 The advantage of a global installation is that you don't have to prefix your commands with npx, yarn or pnpm. The global Nx installation hands off the process execution to the local Nx installation in your repository, which eliminates any issues with outdated globally installed packages.
 
-Learn more about [managing and troubleshooting a global Nx installation](/concepts/more-concepts/global-nx).
+### Updating your global Nx installation
+
+There are some cases where an issue could arise when using an outdated global installation of Nx. If the structure of your Nx workspace no longer matches up with what the globally installed copy of Nx expects, it may fail to hand off to your local installation properly and instead error. This commonly results in errors such as:
+
+- Could not find Nx modules in this workspace.
+- The current directory isn't part of an Nx workspace.
+
+If you find yourself in this position, you will need to update your global install of Nx.
+
+In most cases, you can update a globally installed npm package by rerunning the command you used to install it, as described [above](#installing-nx-globally)
+
+If you cannot remember which package manager you installed Nx globally with or are still encountering issues, you can locate other installs of Nx with these commands:
+
+{% tabs %}
+{% tab label="npm" %}
+
+```shell
+npm list --global nx
+```
+
+{% /tab %}
+{% tab label="yarn" %}
+
+```shell
+yarn global list nx
+```
+
+{% /tab %}
+{% tab label="pnpm" %}
+
+```shell
+pnpm list --global nx
+```
+
+{% /tab %}
+{% /tabs %}
+
+{% callout type="note" title="Older Global Installations" %}
+
+In prior versions, Nx could be installed globally via `@nrwl/cli` or `@nrwl/tao`. If you are seeing these warnings but cannot find other global installations of Nx via the above commands, you should look for these packages as well. In general, you should remove these and install the latest version of `nx` instead.
+
+{% /callout %}
+
+You can then remove the extra global installations by running the following commands for the duplicate installations:
+
+{% tabs %}
+{% tab label="npm" %}
+
+```shell
+npm rm --global nx @nrwl/cli @nrwl/tao
+```
+
+{% /tab %}
+{% tab label="yarn" %}
+
+```shell
+yarn global remove nx @nrwl/cli @nrwl/tao
+```
+
+{% /tab %}
+{% tab label="pnpm" %}
+
+```shell
+pnpm rm --global nx @nrwl/cli @nrwl/tao
+```
+
+{% /tab %}
+{% /tabs %}
+
+Finally, to complete your global installation update, simply reinstall it as described [above](#installing-nx-globally).
+
+## Self-managed Nx installation
```

Comment:
Yep, the "name" of the Nx wrapper or whatever we used to call it is weird and in general hard to find a proper one.

RE moving it to its own recipe. Sure. But it is the same reasoning as mentioned before for the global installation. A new user would maybe look at it and then think, this is not for me, I just want to get it going in my new/existing JS repo. A user in a Java/.net workspace though might think about the weirdness of node_modules and then go search for installation guides on how to avoid that.